js/privatebin.js

```diff
@@ -77,6 +77,13 @@ jQuery.PrivateBin = (function($, RawDeflate) {
         }
     };
 
+    /**
+     * URL fragment prefix requiring load confirmation
+     *
+     * @private
+     */
+    const loadConfirmPrefix = '#?';
```

Comment:
Given the ambiguity such a character is used in URLs for encoding query params etc. I am not sure whether this is the best character to find after the anchor/fragement part? https://en.wikipedia.org/wiki/Percent-encoding
`~` seems to be a safe character?

[But apparently spec allows any ASCII chars](https://url.spec.whatwg.org/#concept-url-fragment)? Just saying this, because nowadays [there can be crazy stuff in there](https://developer.mozilla.org/en-US/docs/Web/Text_fragments).

I have nothing against this, we should just think of potential problems of this schema etc. now, before all URLs are in use… :upside_down_face: 

Ah or did you use that because of the meaning as question mark?

Reply:
This is certainly worth thinking over twice. Yes, I picked the question mark simply because the effect will be to ask the user, so adding it triggers a question. My only other concern was to not use any hexadecimal characters (0-9, a-f or A-F) and to keep it to just one character since a lot of folks don't like long URLs. But I think we could really use any other ASCII character, also upper or lower case letters, punctuation, etc. I would like it to remain a visible character and something not easily lost when copy pasting across constrained media or communication channels. That would be an argument against the question mark, as some parsers may not allow it or think it is where they can append URL arguments after (thinking of certain social media platforms adding tracking parameters). So here are some more ideas:
- example.com/?aaaa**#q**bbbb (as in "question")
- example.com/?aaaa**#Q**bbbb
- example.com/?aaaa**#-**bbbb (small)
- example.com/?aaaa**#>**bbbb
- example.com/?aaaa**#G**bbbb (first non-hex letter)
- example.com/?aaaa**#i**bbbb (least visible, might get us in trouble with a certain fruit company)

Certainly worth pondering some more.

Reply:
I realize my above statement reg. avoiding hexadecimal characters is not correct: In the URL fragment, we use base58 (v2 pastes) or base64 (v1 pastes) encoding. This means we should avoid using a-z, A-Z, 0-9, +, / and = (for base64). Base58 uses a smaller alphabet, but someone could modify an older v1 paste URL to ask the question, so we better avoid all of these. This would restrict us to punctuation like !, dash, underscore, semicolons and colons, brackets, ...

Well, or we go unicode and use an emoji ;-) It would make the URL longer and there are some risks when shared over services that would apply URL encoding to it. So probably best to stick to ASCII.

Reply:
For me a simple `-` or `_` or so would be good IMHO. < and > could cause problems with (Markdown) URL parsers.

Reply:
I've now updated the prefix to a dash (`-`) and also re-fixed the password modal focusing.

Found that it would still loose focus shortly after getting set. Since the debugger pointed into jquery code, not ours, I applied the workaround with a timeout to focus back on it. Happens in Firefox as well as Chrome. :-/ Hope we can drop this when moving on to a newer bootstrap version.